services/controller/run

```diff
@@ -16,7 +16,7 @@ export PYTHONPATH='./src'
 exec gunicorn \
   --bind="0.0.0.0:${PORT}" \
   --workers=16 \
-  --timeout=120 \
+  --timeout=600 \
```

Comment:
Not a great solution but it’ll do for now.

Reply:
I wholeheartedly agree